.github/workflows/nemo_tests.yml

```diff
@@ -183,17 +183,18 @@ jobs:
     # PSyclone, compile and run ECMWF NEMO with OpenMP for CPUs
     - name: NEMO ECMWF OpenMP for CPU
       run: |
-        source .runner_venv/bin/activate
+        . .runner_venv/bin/activate
         export PSYCLONE_NEMO_DIR=${GITHUB_WORKSPACE}/examples/nemo/scripts
         export NEMO_DIR=${HOME}/NEMOGCM_V40
         export COMPILER_ARCH=linux_intel
         export ADD_KEYS="IEEE_IS_NAN=ieee_is_nan key_nosignedzero"
         export DEL_KEYS="key_iomput"
         export MODEL_DIR=/archive/ssiso/ecmwf_eORCA1_GO8/
         export NAMELISTS_DIR=${NEMO_DIR}/testscripts_V40/output/openmp_outer_V40_eORCA1_GO8_Z75_20170906_cray_dp_1x1/
-        cd examples/nemo/scripts
+        cd $PSYCLONE_NEMO_DIR
         make -j 4 openmp_cpu
         module load intel/oneapi compiler mpi
```

Comment:
Has there been any change in the Intel compilers? In https://gist.github.com/sergisiso/a4049a0fc0a0a11651a5ce6a04d76160 it seems you got a free GlaDos and this is the only test which was slower than expect?

I am not asking to revering or fixing anything? Just asking if we know the cause.

Reply:
Ok, no worries if there hasn't been a specific change.